content/js/users.md

```diff
@@ -140,6 +140,11 @@ While each of the sign-up functions is resolved with a user object,
 in most cases you need not deal with it because
 you can access the currently logged-in user using `skygear.currentUser`.
 
+If your app does not require the user to create an account before using your
+app, you can sign up the user anonymously. An user that is signed up
+anonymously does not have a username, email, nor a password. You can add a
+username, email and/or password to a user after signing up.
```

Comment:
- each of these users still has their own user IDs -> YES
- can an anonymous user create records? -> YES
- does an anonymous user have a default role? -> YES

anonymously is just normal user without username and password. And once you lost your access_token, you lost the account forever.